### DISCUSSION TITLE
Why is the `desktop` image enforced?

### DISCUSSION THREAD

**Discussion by andrewvaughan** [Sithuk/ubuntu-server-zfsbootmenu#45] [General]
I find it odd that `ubuntu-desktop` explicitly is enforced - it's rare that I have that ISO in my library, as I'm almost always installing headless servers.

Is there something about the `ubuntu-desktop` ISO that is unique? Does server not have ZFS 2.0 and I just can't find that info anywhere? Just curious - and maybe we can lessen that enforcement (or add to the FAQ) in that case?

**Comment by Sithuk** [comment]
The requirement for the desktop live iso was because of issue 17 (linked below). The desktop and server iso had different apt.sources files. So I picked one so the script only had to support one live iso version. 
https://github.com/Sithuk/ubuntu-server-zfsbootmenu/issues/17
